documentation/create_overview_doc/README.md

```diff
@@ -19,17 +19,17 @@ This powerful tool streamlines the creation of high-level documentation for soft
 
 - **Auto-Retry Logic and Enhanced Error Handling**: Ensures reliable documentation generation even in unstable environments through sophisticated error handling and retry mechanisms.
 
-- **Visualization with Mermaid.js Flow Maps and Enhanced Syntax Handling**: AI-generated visual flow maps to represent module interactions. Now with improved handling of Mermaid diagram syntax, minimizing manual interventions.
-
-- **Mermaid CLI Image Conversion**: Transforms Mermaid.js diagrams into images, offering a clear visual representation of your code architecture.
+- **Visualization with Code2Flow**: AI-generated visual flow maps to represent module interactions, configured with flags such as --hide-legend, --no-trimming, --verbose, and --skip-parse-errors for optimal detail and clarity.
 
+- **Code2Flow CLI Image Conversion**: The tool uses Code2Flow for creating detailed flow maps, offering a clear visual representation of your code architecture.
+- **Customizable Flowchart Generation: Code2Flow** Configured with flags such as --hide-legend, --no-trimming, --verbose, and --skip-parse-errors to optimize the detail and clarity of flowcharts.**
```

Comment:
**Suggestion**: Consider adding examples or links to Code2Flow documentation for the newly introduced flags (--hide-legend, --no-trimming, --verbose, and --skip-parse-errors) to help users understand their usage and impact on flowchart generation. <br> **Code Suggestion**: 
 ```
 - **Customizable Flowchart Generation with Code2Flow**: Enhanced with flags such as --hide-legend, --no-trimming, --verbose, and --skip-parse-errors for optimal detail and clarity. For more information on these flags, refer to [Code2Flow documentation](https://github.com/scottrogowski/code2flow).
 ```